WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -113,28 +119,69 @@ public View onCreateView(final LayoutInflater inflater, ViewGroup container, Bun
     public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
-        mUsernamesContainer = (ViewGroup) getView().findViewById(R.id.usernames);
+        mUsernamesContainer = (ViewGroup) view.findViewById(R.id.usernames);
+        mUsernamesContainer.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                EditTextUtils.showSoftInput(mUsernameEditText);
+            }
+        });
 
         String role = mRole;
         if (role == null) {
             role = loadDefaultRole();
         }
 
-        mUsernameEditText = (EditText) getView().findViewById(R.id.invite_usernames);
+        mUsernameEditText = (MultiUsernameEditText) view.findViewById(R.id.invite_usernames);
+        mUsernameEditText.setText(" ");
+        mUsernameEditText.setSelection(1);
```

Comment:
Hey @khaykov , let'a add a comment here about the setting of the space char and the selection.

Additionally and since this `setText(" ")` is used a few times, what about having it in function even though it's one line of code? I think it would benefit the code readability. Thanks!